charts/lightstepsatellite/README.md

```diff
@@ -100,4 +100,4 @@ Lightstep microsatellite to collect telemetry data.
 | tolerations | list | `[]` |  |
 
 ----------------------------------------------
-Autogenerated from chart metadata using [helm-docs v1.5.0](https://github.com/norwoodj/helm-docs/releases/v1.5.0)
+Autogenerated from chart metadata using [helm-docs v1.11.0](https://github.com/norwoodj/helm-docs/releases/v1.11.0)
```

Comment:
why this change?

Reply:
oh we need to update it here: https://github.com/lightstep/lightstep-microsatellite-helm-chart/blob/main/.github/helm-docs.sh#L4